### ISSUE TITLE
Keyboard nav not working with screen reader on

### ISSUE THREAD

**Issue by mmazanec22** [emeeks/semiotic#277]
For treemap and sankey, tooltip does not follow focus when VoiceOver is on with either Chrome or Safari.  For bar charts, keyboard nav doesn't work at all when the screen reader is on-- skips right to next element.

**Comment by mmazanec22** [comment]
This appears to be related to my use of the img role on a parent element, and not to semiotic at all.  Sorry for the trouble and thanks for looking into it!
